INSTALL.md

```diff
@@ -213,62 +237,76 @@ filling in this section of the README.
 
 ### Linux / Windows
 
-Once all the prerequisites are installed, compiling Alacritty should be easy:
-
 ```sh
 cargo build --release
 ```
 
 If all goes well, this should place a binary at `target/release/alacritty`.
 On Windows this directory should also contain the `winpty-agent.exe`.
 
-#### Desktop Entry
-
-Many linux distributions support desktop entries for adding applications to
-system menus. To install the desktop entry for Alacritty, run
+### MacOS
```

Comment:
```suggestion
### macOS
```

Reply:
I'm also curious why macOS gets a full build (not just the binary) while the other platforms have additional components through the Post Build. I feel like the whole Building section could remove the platform subsections, altogether.

Reply:
I'm not sure what you mean?

Reply:
Like an "App" build should be self-contained (or at least to the best of it's ability), which includes things like the icon. Futhermore, `make app` itself is more or less the macOS equivalent of the whole Post Build section, with the exception of not actually installing things. It would be nice if the man pages, and terminfo files could be served directly from the App bundle itself, but to my knowledge, that's not really an option.

So I just think the process of building on macOS is basically the same as Linux/Windows, it's the Post Build where whole `Makefile` seems most relevant.

Reply:
But you don't want to build like on linux? You want to `make app`? Why would you `cargo build --release` just to `make app` which calls `cargo build --release` again. I don't understand what your goal is.

Reply:
I guess this is all in the "Manual Installation" section, so I can see why it makes sense to just take the shortest path to that end for macOS. That said, `make app` does not install the man page for example. So I guess macOS is just in a bit of a strange situation here. It should all be fine though, since the Post Build instructions should still apply, and the files packaged into the App bundle are mostly there in case someone doesn't have access to the source, which is where this INSTALL.md is located.

Basically, perhaps it could be made more clear, but I see no issue with this now.

Reply:
> That said, make app does not install the man page for example. So I guess macOS is just in a bit of a strange situation here.

Yes, that's why I moved the post build instructions to a completely separate step. Getting the auxiliary files from the app bundle instead of the repository when you're already inside it is just more work than it's worth. So easier to just document it the same for all platforms.